test_unstructured_ingest/test-ingest-google-drive.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+cd "$SCRIPT_DIR"/.. || exit 1
+
+if [ -z "$GCP_INGEST_SERVICE_KEY" ]; then
+   echo "Skipping Google Drive ingest test because the GCP_INGEST_SERVICE_KEY env var is not set."
+   exit 0
+fi
```

Comment:
Might as well include the google drive link here to the folder to make it easy to find?